app/controllers/api/v1/groups/relationships/machines_controller.rb

```diff
@@ -10,7 +10,7 @@ class MachinesController < Api::V1::BaseController
     authorize :group
 
     def index
-      machines = apply_pagination(authorized_scope(apply_scopes(group.machines), with: Groups::MachinePolicy).preload(:product, :policy, :owner, license: %i[owner]))
+      machines = apply_pagination(authorized_scope(apply_scopes(group.machines), with: Groups::MachinePolicy).preload(:product, :policy, :owner, license: %i[policy owner]))
```

Comment:
Wish there was a way to tell active record that `machine.policy == license.policy`…